apps/camera/js/controllers/zoom-bar.js

```diff
@@ -55,11 +62,16 @@ ZoomBarController.prototype.onChange = function(value) {
 };
 
 ZoomBarController.prototype.onZoomConfigured = function(zoom) {
+  this.configureZoom();
```

Comment:
nit: It seems strange to call a method called `configureZoom()` inside a method called `onZoomConfigured()` --  Perhaps `configureZoom()` should be called `configureZoomBar()`?